reducers/storage.js

```diff
@@ -49,12 +49,14 @@ function storage(state = {}, action) {
         return nextState;
     }
     case StorageTypes.CLEAR: {
-        var cleanState = {};
-        action.data.exclude.forEach((excluded) => {
-            if (state[excluded]) {
-                cleanState[excluded] = state[excluded];
-            }
-        });
+        const cleanState = {};
+        if (action.data && action.data.exclude && action.data.exclude.forEach) {
+            action.data.exclude.forEach((excluded) => {
```

Comment:
@lieut-data Again on an unrelated note: Does exclude in the payload persist data in indexDb on logout?. Last time i checked this does not work as we purge everything on logout so we had to use localStorage for any persisting data.

Reply:
I confess I'm not sure what the backing is for this reducer. All of this code seems to have gone a bit stale, and I suspect I could just rip out the `excluded` parameter altogether given that no one (now) appears to be using it.

We're definitely continuing to use `localStorage` for anything that must remain past logout.